rectangles.js

```diff
@@ -0,0 +1,25 @@
+function area(height, width) {
+  return height * width
+}
+
+function perimeter(height, width) {
+  return (height * 2) + (width * 2)
+}
+
+function diagonal(height, width) {
+  heightSquared = Math.pow(height, 2)
+  widthSquared = Math.pow(width, 2)
```

Comment:
since you are doing the same thing to both the width and height, you could even make a `square` method that does this logic for you 👍